shared/reducers/chat2.tsx

```diff
@@ -893,6 +893,28 @@ const reducer = Container.makeReducer<Actions, Types.State>(initialState, {
       })
     })
 
+    // Identify convs that have a zzz CHANNEL_INACTIVE journeycard.
+    // Convs that used to have a zzz journeycard and just got a newer message should delete the journeycard.
+    Object.keys(convoToMessages).forEach(cid => {
+      const messages = convoToMessages[cid]
+      const jc = messages.find(
+        m => m.type == 'journeycard' && m.cardType == RPCChatTypes.JourneycardType.channelInactive
+      ) as Types.MessageJourneycard | undefined
+      if (jc) {
+        draftState.hasZzzJourneycard.set(cid, jc)
+        draftState.shouldDeleteZzzJourneycard.delete(cid)
+      } else {
+        const priorJc = draftState.hasZzzJourneycard.get(cid)
+        if (priorJc) {
+          // Find a message that has a later ordinal and so should cause the zzz to disappear.
+          if (messages.find(m => m.ordinal > priorJc.ordinal)) {
```

Comment:
`messages.some(...)`

Reply:
@mlsteele `some` returns a boolean which is probably what you want rather than relying on the truthiness of whatever the message value is.